ADCore/ADCore.yml

```diff
@@ -0,0 +1,85 @@
+module: ADCore
```

Comment:
Any possiblity to make a schema for these files? Also, could we put a different file extensions on it to distinguish it from `ADCore.ibek.support.yaml`?

Reply:
Agreed on both.

Reply:
ADCore.install.yml ?